rel/overlay/etc/default.ini

```diff
@@ -208,6 +208,11 @@ bind_address = 127.0.0.1
 ;
 ; Bulk docs transaction batch size in bytes
 ;update_docs_batch_size = 2500000
+;
+; The upper bound, in milliseconds, that a continuous changes feed will return
+; database results for. The response ends on receipt of the first change to occur
+; after the duration period has lapsed.
+;changes_duration =
```

Comment:
It's a bit odd not to have a value. Would "infinity" work here as a default?

Reply:
It would not, because that would be a list rather than `undefined`